package.json

```diff
@@ -23,15 +22,18 @@
     "test:travis": "polymer test --config-file wct.conf-travis.json"
   },
   "files": [
-    "/cli",
     "/components",
     "/directives",
+    "/generated",
     "/helpers",
     "/mixins",
     "/tools",
     "!demo",
+    "!lang",
+    "!screenshots",
     "!test",
     "/components/demo",
+    "/generated/lang",
```

Comment:
Why don't we have `/generated/icons` as well?

Reply:
So all of `/generated` is included. But then I wanted stuff inside `/components/whatever/lang` to be _excluded_ so I added `!lang`, but then that excludes `/generated/lang` so I explicitly _included_ it. Haha if you can think of a cleaner way I'm all 👂 s.

Reply:
Oh! that makes sense thanks :) Sadly, don't know a better way.

Reply:
An alternative approach I think may be to use `files` for inclusions, and then also use `.eslintignore` to exclude things within those.  I did not try it, and I am not convinced it's any better than this because it's basically the same, except spread across two different files, which isn't great.  That article you linked to me a while ago suggested that was the "one and only" time that `.eslintignore` would be "ok".

Reply:
Assuming you mean `.npmignore`?

Reply:
Haha sorry yep!